reference-implementation/lib/readable-stream.js

```diff
@@ -488,23 +488,20 @@ function create_ReadableStreamTeePullFunction() {
         return;
       }
 
+      const value1 = value;
+      const value2 = value;
+
       // There is no way to access the cloning code right now in the reference implementation.
       // If we add one then we'll need an implementation for StructuredClone.
-
+      // if (teeState.canceled2 === false && cloneForBranch2 === true) {
+      //   value2 = StructuredClone(value2);
+      // }
```

Comment:
Even when the branch 1 has been cancelled, we clone values. I think it's ok and is more consistent, but leaving this comment to make sure you're aware of.